.jenkins/build_docker_images.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Set variable to allow parallel building in the Docker image creation
+export DOCKER_BUILDKIT=1
+
+# Get the dockerfiles for the images to be tested on Piz Daint
+dockerfiles=( $( ls docker/Dockerfile.* ) )
+
+for df in ${dockerfiles[@]}; do
+   # Grab the architecture tag from the dockerfile filename
+   arch=${df:18}
+
+   # Set the names of the new Docker image and tar file
+   container=us.gcr.io/vcm-ml/fv3gfs-compiled:$arch
+   tar_file=fv3gfs-compiled_${arch}.tar
+   
+   # Build the docker image and push image to VCM's Google Container Repository
+   docker build -f $df --target fv3-bld -t $container .
+   docker push $container
+
+   # Copy an archived version of the image to a public Google Storage Bucket
+   docker save $container -o $tar_file
+   gzip $tar_file
+   gsutil copy ${tar_file}.gz gs://vcm-ml-public/jenkins-tmp/${tar_file}.gz
```

Comment:
Instead of saving a tar file for upload/download, can we `docker pull`, `docker save`, then `sarus load` in the sarus test, instead of downloading the tar file? Or is there a reason this is better? I'm concerned that this way means we're uploading two copies of every image to the cloud.

Reply:
There is no docker running on Piz Daint -we have to use Sarus.  Unfortunately Sarus cannot pull directly from GCR due to it authentication process.  The docker save/tar file process is a workaround.

BTW. Singularity can perform a pull directly from GCR but unfortunately it doesn't seem to work on the GPU nodes of Daint currently.

Reply:
So we could stop the docker push currently to GCR -but others may want access to these Docker images for running on a cloud VM